Program.cs

```diff
@@ -1,17 +1,37 @@
 ﻿using MongoDB.Driver;
 using MongoDB.Bson;
+using System.Diagnostics;
 
 public static class Example
 {
     public static void Main(string[] args)
     {
-        var client = new MongoClient(args[0].ToString());
+        var connectionUri = args[0].ToString();
+        var settings = MongoClientSettings.FromConnectionString(connectionUri);
+        var client = new MongoClient(settings);
 
         IMongoDatabase db = client.GetDatabase("test");
         var command = new BsonDocument { { "ping", 1 } };
         var res = db.RunCommand<BsonDocument>(command);
+        Debug.Assert(res == new BsonDocument { { "ok", 1.0 } }, "ping failed");
 
-        Console.WriteLine(res);
+        command = new BsonDocument { { "dropDatabase", 1 } };
+        res = db.RunCommand<BsonDocument>(command);
+        Debug.Assert(res == new BsonDocument { { "ok", 1.0 } }, "dropDatabase failed");
+
+        var documentList = new List<BsonDocument>{
+            new BsonDocument { { "_id", 1 }, { "a", 1 } },
+            new BsonDocument { { "_id", 2 }, { "a", 2 } },
+            new BsonDocument { { "_id", 3 }, { "a", 3 } },
+            new BsonDocument { { "_id", 4 }, { "a", 4 } },
+        };
+
+        var collection = db.GetCollection<BsonDocument>("foo");
+        collection.InsertMany(documentList);
+
+        var filter = Builders<BsonDocument>.Filter.Eq("a", 4);
+        BsonDocument actual = collection.Find(filter).FirstOrDefault();

```

Comment:
The python example uses `findone`, on the other hand java and c# use `find`, could we keep that consistent between examples please? :hugs:

Reply:
See [above](https://github.com/FerretDB/dotnet-example/pull/5#discussion_r1290065299) comment, there's no `FindOne` method AFAIK.

Reply:
Ah, for languages that can do filtering like linq doesn't provide fineone I see.